http/client/portainer_client.go

```diff
@@ -34,39 +34,40 @@ func NewPortainerClient(serverAddress, endpointID, edgeID string) *PortainerClie
 }
 
 type stackConfigResponse struct {
+	Name string
 	StackFileContent string
 	Prune            bool
 }
 
 // GetEdgeStackConfig retrieves the configuration associated to an Edge stack
-func (client *PortainerClient) GetEdgeStackConfig(edgeStackID int) (string, bool, error) {
+func (client *PortainerClient) GetEdgeStackConfig(edgeStackID int) (string, string, bool, error) {
```

Comment:
3 return values are OK in some cases but should be exceptional.

4+ should never happen, please introduce a structure that you'll use here instead.